livingdoc-fixture-converter/src/test/java/org/livingdoc/fixture/converter/number/NumberConverterContract.kt

```diff
@@ -0,0 +1,121 @@
+package org.livingdoc.fixture.converter.number
+
+import com.nhaarman.mockito_kotlin.mock
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Assertions.assertThrows
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.livingdoc.fixture.api.converter.ConversionException
+import org.livingdoc.fixture.api.converter.Language
+import org.livingdoc.fixture.api.converter.TypeConverter
+import org.mockito.BDDMockito.given
+import utils.EnglishDefaultLocale
+import java.lang.reflect.AnnotatedElement
+
+@EnglishDefaultLocale
+internal abstract class NumberConverterContract<T : Number> {
+
+    abstract fun getCut(): TypeConverter<T>
+
+    abstract fun getMinValue(): T
```

Comment:
These getters all return constants. Would feel more like Kotlin to me to turn them into read-only properties.